docs_user/modules/proc_migrating-mon-from-controller-nodes.adoc

```diff
@@ -315,35 +291,34 @@ placement:
 unmanaged: true
 ----
 
-. Save the spec in the `/tmp/mon.yaml` file
-. Apply the spec with cephadm by using the orchestrator:
+. Save the spec in the `/tmp/mon.yaml` file.
+
+. Apply the spec with `cephadm` by using the Ceph Orchestrator:
 +
 ----
 $ sudo cephadm shell -m /tmp/mon.yaml
 $ ceph orch apply -i /mnt/mon.yaml
 ----
 +
-The mon daemons are marked as `<unmanaged>`, and it is now possible to redeploy
-the existing daemon and bind it to the migrated IP address.
+The Ceph Monitor daemons are marked as `unmanaged`, and you can now redeploy the existing daemon and bind it to the migrated IP address.
 
-. Delete the existing mon on the target node:
+. Delete the existing Ceph Monitor on the target node:
 +
 ----
 $ ceph orch daemon add rm mon.<target_node> --force
 ----
 +
-. Redeploy the new mon on the target using the old IP address:
+* Replace `<target_node>` with the hostname of the target node that is included in the {Ceph} cluster.
+
+. Redeploy the new Ceph Monitor on the target node by using the previous IP address:
```

Comment:
I changed "old" to previous. I assume that's what was meant here. But this is confusing because the step mentions the "previous IP address" while the text below the step mentions the "migrated IP address:"
Step: . "Redeploy the new Ceph Monitor on the target node by using the previous IP address:"
...
Text below step: "Replace `<ip_address>` with the IP address of the migrated IP address." 


Can you help clarify?